transformer_engine/pytorch/transformer.py

```diff
@@ -13,7 +13,7 @@
 
 from flash_attn.flash_attn_interface import flash_attn_unpadded_func
 
-from transformer_engine.pytorch import LayerNormLinear, Linear, LayerNormMLP, LayerNorm
+from transformer_engine.pytorch.module import LayerNormLinear, Linear, LayerNormMLP, LayerNorm
```

Comment:
why is this needed?

Reply:
Without it autoapi complained about the circular reference since te.pytorch imports stuff from the.pytorch.transformer which then imports things from te.pytorch. I believe we were just lucky before with the order with which we do stuff in __init__.py but we should not rely on it.